pyriemann/utils/viz.py

```diff
@@ -44,32 +48,51 @@ def plot_embedding(X,
     .. versionadded:: 0.2.6
     """
     if embd_type == "Spectral":
-        lapl = SpectralEmbedding(n_components=2, metric=metric)
+        e = SpectralEmbedding(n_components=2, metric=metric)
     elif embd_type == "LocallyLinear":
-        lapl = LocallyLinearEmbedding(n_components=2,
+        e = LocallyLinearEmbedding(n_components=2,
                                       n_neighbors=X.shape[1],
                                       metric=metric)
+    elif embd_type == "TSNE":
+        e = TSNE(n_components=2, metric=metric)
```

Comment:
The example takes a looong time to build...
```suggestion
        e = TSNE(n_components=2, metric=metric, max_iter=1000)
```

Reply:
I think we should start with smaller number of electrodes for the t-SNE in the example. That being said, it is a good idea to introduce a default limit for the max iteration.